ansible/inventory/inventory

```diff
@@ -0,0 +1,20 @@
+[microshift]
+microshift.example.com ansible_host= private_ip= # microshift node
+
+[microshift:vars]
+ansible_user=centos
```

Comment:
Can we use `microshift` user here?

Reply:
Does the user depend on the image?

Reply:
Yeah it does, the AMIs for CentOS have user `centos`. It is a var after all so we can set it as necessary.
Once I move to image builder AMIs we can make the default user `microshift`.